docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,92 @@
+# Services removal
+
+We removed services (i.e. *Service classes - not to be interchanged with Symfony services) from our model and moved logic into more suitable places.
+Following instructions tell you which method was moved where, so you can upgrade your code and also tests.
+
+If you use these methods, change their calling appropriately:
+- `CustomerService::edit(User $user, UserData $userData)`  
+  ⮕ `User::edit(UserData $userData, CustomerPasswordService $customerPasswordService)`
```

Comment:
There are weird characters and I do not like them...:-)

Reply:
ok, soo.... what is wrong with them? what do you suggest?